### ISSUE TITLE
Parsing fails for triple quoted strings that extend over newlines.

### ISSUE THREAD

**Issue by fkearney8** [amznlabs/ion-intellij-plugin#2]
Examples like:

{
  some_value: "foo",
  doc: '''Here is some sample text that has triple quotes and'''
          ''' extends over a newline is supposed to work, but the plugin doesn't like it.''',
  other_value: 1 
}

...aren't processed properly by the plugin. It says a comma or brace is expected.

**Comment by belloale** [comment]
Is this a valid multi-line string? Not sure if this is in the spec anywhere.

**Comment by tgregg** [comment]
@belloale Yes, it's valid: https://amzn.github.io/ion-docs/docs/spec.html#string
